### ISSUE TITLE
New ZIM request: raspberrypi.com/documentation

### ISSUE THREAD

**Issue by thatsitipl** [openzim/zim-requests#467]
- Website URL: https://www.raspberrypi.com/documentation/
- License: CC-by-sa 
- Desired ZIM Title: Rapsberry Pi Docs
- Desired ZIM Description: The official documentation for Raspberry Pi computers and microcontrollers
- Desired ZIM Icon –png (URL or attach one): https://www.raspberrypi.com/app/uploads/2021/10/cropped-Raspberry-Pi-Favicon-100x100-1-32x32.png
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?: **no**

The Repository is available at : https://github.com/raspberrypi/documentation

**Comment by RavanJAltaie** [comment]
Succeeded.

**Comment by Popolechien** [comment]
It has not appeared on the library yet, and I can't find in in the dev library either. Do you have a link (not to the zim file itself, but to its demo on the library)

**Comment by RavanJAltaie** [comment]
No, I checked as well, the recipe succeeded but it didn't reflected to the library

**Comment by benoit74** [comment]
Recipe moved to https://farm.openzim.org/recipes/raspberrypi.com_en_documentation, fixed and requested

**Comment by Jaifroid** [comment]
Thank you @benoit74  for re-activating this. So I've tested the resulting ZIM (https://dev.library.kiwix.org/viewer#raspberrypi.com_en_documentation_2025-11), and unfortunately it appears that all the pages Browsertrix attempted to navigate to have been caught by `challenges.cloudflare.com` and present as external links. Look at the URLs:

<img width="1212" height="1540" alt="Image" src="https://github.com/user-attachments/assets/62e9c588-ae11-4073-b1b2-d8880fa600ce" />

The PWA has a utility for viewing the URL index easily: type "` /`" in search bar [space followed by forward slash]. Under the C namespace there are pages and pages of either `challenges.cloudflare.com` or `www.raspberrypi.com/cdn-cgi/challenge-platform` which must be related.

You can also see the title index by typing just a space in the PWA search bar, and you'll see the vast majority of pages are entitled "Checking your Browser".

This isn't the first time I've seen this issue, I also saw it in the cloudflare learning centre ZIM that doesn't work in Chromium browsers.

**Comment by benoit74** [comment]
Note sure there is much we can do unfortunately at this stage. I would hope https://github.com/kiwix/operations/issues/373 could help, but it has not been judged a priority so far.

**Comment by Jaifroid** [comment]
Ah, there is a similar issue on mwOffliner https://github.com/openzim/mwoffliner/issues/2134. Should clearly be higher priority, if it is affecting core sites as well as useful documentation sites like the one in this issue.

**Comment by Popolechien** [comment]
I can only concur.
